pkg/adapter/v2/cloudevents.go

```diff
@@ -113,6 +116,24 @@ type client struct {
 
 var _ cloudevents.Client = (*client)(nil)
 
+// SendCloudEvent prepares logging while sending an event for APIServerSource and PingSource
+// and sends the event to client.Send
+func SendCloudEvent(ctx context.Context, event cloudevents.Event, logger *zap.SugaredLogger, ce cloudevents.Client) {
+	event = event.Clone()
+	event.SetID(uuid.New().String()) // provide an ID here so we can track it with logging
+	defer logger.Debug("Finished sending cloudevent id: ", event.ID())
+	source := event.Context.GetSource()
+	subject := event.Context.GetSubject()
+	logger.Debugf("sending cloudevent id: %s, source: %s, subject: %s", event.ID(), source, subject)
```

Comment:
Please keep it the way it was before by printing the target

Reply:
I changed it to eventContext + target, reference: https://knative.slack.com/archives/C9JP909F0/p1611825333052400